### PR TITLE
LTR 10 Cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/aragorn_the_unifier.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Name:Aragorn, the Uniter
+ManaCost:R G W U
+Types:Legendary Creature Human Noble
+PT:5/5
+T:Mode$ SpellCast | ValidCard$ Card.White | ValidActivatingPlayer$ You | TriggerZones$ Battlefield |  Execute$ TrigToken | TriggerDescription$ Whenever you cast a white spell, create a 1/1 white Human Soldier creature token.
+T:Mode$ SpellCast | ValidCard$ Card.Blue | ValidActivatingPlayer$ You | TriggerZones$ Battlefield |  Execute$ TrigScry | TriggerDescription$ Whenever you cast a blue spell, scry 2.
+T:Mode$ SpellCast | ValidCard$ Card.Red  | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigDamage | TriggerDescription$ Whenever you cast a red spell, CARDNAME deals 3 damage to target opponent.
+T:Mode$ SpellCast | ValidCard$ Card.Green| ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigPump | TriggerDescription$ Whenever you cast a green spell, target creature gets +4/+4 until end of turn.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/aragorn_the_unifier.txt:8]: spaces before "|" Red has one too many, green does miss one

File: forge-gui/res/cardsfolder/upcoming/aragorn_the_unifier.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Name:Aragorn, the Uniter
+ManaCost:R G W U
+Types:Legendary Creature Human Noble
+PT:5/5
+T:Mode$ SpellCast | ValidCard$ Card.White | ValidActivatingPlayer$ You | TriggerZones$ Battlefield |  Execute$ TrigToken | TriggerDescription$ Whenever you cast a white spell, create a 1/1 white Human Soldier creature token.
+T:Mode$ SpellCast | ValidCard$ Card.Blue | ValidActivatingPlayer$ You | TriggerZones$ Battlefield |  Execute$ TrigScry | TriggerDescription$ Whenever you cast a blue spell, scry 2.
+T:Mode$ SpellCast | ValidCard$ Card.Red  | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigDamage | TriggerDescription$ Whenever you cast a red spell, CARDNAME deals 3 damage to target opponent.
+T:Mode$ SpellCast | ValidCard$ Card.Green| ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigPump | TriggerDescription$ Whenever you cast a green spell, target creature gets +4/+4 until end of turn.
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ w_1_1_human_soldier | TokenOwner$ You
+SVar:TrigScry:DB$ Scry| | ScryNum$ 2
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/aragorn_the_unifier.txt:10]: "| |"

File: forge-gui/res/cardsfolder/upcoming/eomer_marshal_of_rohan.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Name:Eomer, Marshal of Rohan
+ManaCost:2 R R
+Types:Legendary Creature Human Knight
+PT:4/4
+K:Haste
+T:Mode$ ChangesZoneAll | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Creature.attacking+Legendary | TriggerZones$ Battlefield | Execute$ TrigUntapAll | ActivationLimit$ 1 | TriggerDescription$ Whenever one or more other attacking legendary creatures you control die, untap all creatures you control. After this phase, there is an additional combat phase. This ability triggers only once each turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/eomer_marshal_of_rohan.txt:6]: Wrong Valid

File: forge-gui/res/cardsfolder/upcoming/eowyn_fearless_knight.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Name:Eowyn, Fearless Knight
+ManaCost:2 R W
+Types:Legendary Creature Human Knight
+PT:3/4
+K:Haste
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigExile | TriggerDescription$ When CARDNAME enters the battlefield, exile target creature an opponent controls with greater power. Legendary creatures you control gain protection from each of that creature's colors until end of turn.
+SVar:TrigExile:DB$ ChangeZone | Origin$ Battlefield | Destination$ Exile | ValidTgts$ Creature.OppCtrl+powerGEX | TgtPrompt$ Select target creature an opponent controls with greater power | RememberChanged$ True | SubAbility$ DBProtection
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/eowyn_fearless_knight.txt:7]: GT

Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/eowyn_fearless_knight.txt:7]: Remember not used

**Review by tool4ever — APPROVED** [review]

**Pull request merged**